.travis.yml

```diff
@@ -1,10 +1,5 @@
 language: node_js
-dist: trusty
-addons:
-  apt:
-    packages:
-      # This is required to run new chrome on old trusty
-      - libnss3
```

Comment:
Why do you remove libnss3? It is needed for puppeteer... 
https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-on-travis-ci

Reply:
I just want to try if installing the package is still necessary on Ubuntu xenial, which is worth trying when needed "on old trusty", which we don't use anymore. When CI greed -> good, when CI red -> revert the commit.

Reply:
All e2e test running without the manuall installation of libnss3 :)